raft-tech/TANF-app
# feat/1787-site-map-rework

## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #1787 

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/, on the spash page there will be no sitemap
2. Sign in and the sitemap link will appear at the bottom.
3. Go to site map and test the links.
4. Accounts with admin access will see the admin page link in the site map.
5. Accounts without admin access will not.

![Screen Shot 2022-08-11 at 9 55 10 AM](https://user-images.githubusercontent.com/7119690/184191119-dc2d32fa-2497-4465-b63c-6c1962fea3a3.png)
![Screen Shot 2022-08-11 at 9 55 17 AM](https://user-images.githubusercontent.com/7119690/184191152-b35d745e-23f0-4a1f-ae06-53b53f275aee.png)
![Screen Shot 2022-08-11 at 9 55 29 AM](https://user-images.githubusercontent.com/7119690/184191166-a3c605e4-c43e-4248-9772-b05f0a0e4d3f.png)
